packages/xstate-vue/test/vue-shim.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.vue' {
```

Comment:
doesn't vue come with types? If it does then we should just install & use them.

Reply:
It does and according to the [docs on Typescript](https://vuejs.org/v2/guide/typescript.html#Official-Declaration-in-NPM-Packages) they should work out of the box, but I couldn't get the tests working without that.